docs/CHILI-GraFx/guides/setup-fsso/index.md

```diff
@@ -1,10 +1,10 @@
-# Setup Federated Single sign on
+# Setup Federated Single Sign On
```

Comment:
I would change page name in Table of Contents to to ` Setup Federated Single Sign On (FSSO)`
1. right now, there is no continuity between table of contents and page itself - titles are different (sure they mean same thing, but you need to know that)
2. having it shortened in title makes it easier to understand in pages what FSSO is - makes it easier to understand what the shortcut means in docs